templates/wordpress/docker-compose.yml

```diff
@@ -0,0 +1,160 @@
+# Use this docker-compose file to host the site locally
+version: "3.7"
+services:
+  nginx:
+    container_name: ${PROJECT_NAME}
+    image: ${NGINX_CONTAINER}
+    expose:
+      - 80
+      - 9222
+    depends_on:
+      - php
+      - mysql
+    networks:
+      - demigod
+    volumes:
+      - ".:/var/www"
```

Comment:
Two things here: 1. Does the nginx virtual host config need to be different for WP than it does for durple? If so, we could inject it.  2. We need some way of sniffing to make sure WP and durple have `webroot true` in their pantheon.yml.

Reply:
FYI: https://github.com/demigod-tools/demigods-nginx/tree/master/nginx

Reply:
this docker-compose is really just a draft and mostly copy-pasta of the drupal one. a WordPress specific docker-compose (and other template files) will come in #3